Update migration metrics copy

renatho opened the pull request:
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to https://github.com/Automattic/wp-calypso/issues/91732

## Proposed Changes

* It updates the metrics copy during the migration flow.
* It also updates the numbers from https://cwvtech.report/ to the last ones (May 2024).

See the following screenshots. Notice that they are:
- Default: When the site is already optimized.
- Dynamic: When WordPress.com hosting could offer better results to their site.

#### Before (Default / Dynamic):

<img width="348" alt="Screenshot 2024-06-21 at 12 12 00" src="https://github.com/Automattic/wp-calypso/assets/876340/b716464f-45a5-4d40-8b32-ac5135ba9e27">
<img width="348" alt="Screenshot 2024-06-21 at 12 12 11" src="https://github.com/Automattic/wp-calypso/assets/876340/931870e9-71ae-41b2-a857-8990d1a8fb99">

#### After (Default / Dynamic):

<img width="348" alt="Screenshot 2024-06-21 at 12 04 10" src="https://github.com/Automattic/wp-calypso/assets/876340/5f828751-bd3c-4546-afad-972a0fa42794">
<img width="348" alt="Screenshot 2024-06-21 at 12 05 37" src="https://github.com/Automattic/wp-calypso/assets/876340/bf7fd1ce-6af5-4fab-a84f-aaa1c1047fb4">


## Why are these changes being made?
<!--
It's easy to see what a PR does but much harder to find out why it was made,
particularly when researching old changes in history. Record an explanation of
the motivation behind this change and how it will help.
-->

* We want to improve the copy for the users.

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Check that the texts look good.
* If you wanna test it running the site, you need to go through the migration flow until the upgrade step (`/setup/site-migration/site-migration-upgrade-plan?from={FROM_SITE}&siteSlug={SITE_SLUG}&siteId={SITE_ID`).

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

renatho committed:
Update cwvtech-report with data from May

renatho committed:
Update migration metrics copy

donnapep reviewed (approved):
Looks good. 👍🏻  Judging by the code, it looks like you're planning to add the "String Freeze" label?

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/15114508

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @renatho for including a screenshot in the description! This is really helpful for our translators.

donnapep commented:
@renatho Should this be blocked?

renatho commented:
> Should this be blocked?

@donnapep, yes! But instead, I just removed it from the board since I had added the issue and the PR there. Now it's only the issue. 😉

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/15114508).

donnapep commented:
@renatho Can you check that the translations are there prior to merging? I only see 3 strings as part of the translation but there should be more.

renatho commented:
@donnapep, good catch!

It seems it's not ready yet. Navigating through GlotPress, in "My Deliverables", I found this other link: https://translate.wordpress.com/deliverables/overview/13555535/

It lists the other missed texts from https://github.com/Automattic/wp-calypso/pull/91818#issuecomment-2191359253 and for some reason the removed strings too. But we can see there that the new ones don't have translation for Spanish yet.

The link from the GitHub actions seems a little more accurate: https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/migration-metrics-copy

donnapep commented:
@renatho Looks like this one is ready to be deployed now. 🥳

renatho commented:
> @renatho Looks like this one is ready to be deployed now. 🥳

@donnapep, it says [here](https://translate.wordpress.com/deliverables/overview/13555535/) that the ES translations are not ready yet for some strings. Did you check a different link?
